test/e2e/specs/site-editor/zoom-out.spec.js

```diff
@@ -3,65 +3,26 @@
  */
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
-// The test is flaky and fails almost consistently.
-// See: https://github.com/WordPress/gutenberg/issues/61806.
-// eslint-disable-next-line playwright/no-skipped-test
-test.describe.skip( 'Zoom Out', () => {
+test.describe( 'Zoom Out', () => {
 	test.beforeAll( async ( { requestUtils } ) => {
-		await requestUtils.activateTheme( 'emptytheme' );
-	} );
-
-	test.beforeEach( async ( { admin, editor, page } ) => {
-		await admin.visitAdminPage( 'admin.php', 'page=gutenberg-experiments' );
-
-		const zoomedOutCheckbox = page.getByLabel(
-			'Enable zoomed out view when selecting a pattern category in the main inserter.'
-		);
-
-		await zoomedOutCheckbox.setChecked( true );
-		await expect( zoomedOutCheckbox ).toBeChecked();
-		await page.getByRole( 'button', { name: 'Save Changes' } ).click();
-
-		await admin.visitSiteEditor();
-		await editor.canvas.locator( 'body' ).click();
-	} );
-
-	test.afterEach( async ( { admin, page } ) => {
-		await admin.visitAdminPage( 'admin.php', 'page=gutenberg-experiments' );
-		const zoomedOutCheckbox = page.getByLabel(
-			'Enable zoomed out view when selecting a pattern category in the main inserter.'
-		);
-		await zoomedOutCheckbox.setChecked( false );
-		await expect( zoomedOutCheckbox ).not.toBeChecked();
-		await page.getByRole( 'button', { name: 'Save Changes' } ).click();
+		await requestUtils.activateTheme( 'twentytwentyfour' );
 	} );
 
 	test.afterAll( async ( { requestUtils } ) => {
 		await requestUtils.activateTheme( 'twentytwentyone' );
 	} );
 
-	test( 'Clicking on inserter while on zoom-out should open the patterns tab on the inserter', async ( {
-		page,
-	} ) => {
-		// Trigger zoom out on Global Styles because there the inserter is not open.
-		await page.getByRole( 'button', { name: 'Styles' } ).click();
-		await page.getByRole( 'button', { name: 'Browse styles' } ).click();
-
-		// select the 1st pattern
-		await page
-			.frameLocator( 'iframe[name="editor-canvas"]' )
-			.locator( 'header' )
-			.click();
-
-		await expect( page.getByLabel( 'Add pattern' ) ).toHaveCount( 3 );
-		await page.getByLabel( 'Add pattern' ).first().click();
-		await expect( page.getByLabel( 'Add pattern' ) ).toHaveCount( 2 );
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.visitSiteEditor();
+		await editor.canvas.locator( 'body' ).click();
+	} );
 
+	test( 'Entering zoomed out mode zooms the canvas', async ( { page } ) => {
+		await page.getByLabel( 'Zoom Out' ).click();
 		await expect(
 			page
-				.locator( '#tabs-2-allPatterns-view div' )
-				.filter( { hasText: 'All' } )
-				.nth( 1 )
-		).toBeVisible();
+				.frameLocator( 'iframe[name="editor-canvas"]' )
+				.locator( 'html' )
+		).toHaveCSS( 'transform', 'matrix(0.75, 0, 0, 0.75, 0, 0)' );
```

Comment:
This is what I thought instintually was the best way to test it. It's dependent on the size of the viewport, I guess, but it should be stable unless the testing setup changes. Please comment if this seems like the wrong approach

Reply:
Yeah, that the content was actually scaled is a tricky thing to test. This is nice and simple and easy to maintain though, and we can keep an eye on the test stability and if it proves flaky try some other approaches.

I think we can also expand it with a couple of extra tests, but they can be in follow-up PRs:
- The canvas is physically smaller after zoomed out is enabled than it was before
- The canvas has a 'frame' around it (it's left, right, top and bottom edges aren't up right next to the viewport area)

Hopefully playwright's `element.boundingBox()` works for these, but I don't know whether it returns scaled values or not.

Reply:
I wanted to do a first PR that just cleans up and removes the skip and then open more for more tests. 

> The canvas is physically smaller after zoomed out is enabled than it was before

Wouldn't the scale CSS imply that this is the truth? 

I think both of those can be part of this test if we want to check that